docs/getting-help/contact.md

```diff
@@ -0,0 +1,22 @@
+# Contacting ICDS Support
+
+If you need help using Roar (RC or RR), please submit your support request **one of the following ways**:
+
+- **Email**: <icds@psu.edu>
+- **Service Portal**: [Submit a ticket via ServiceNow](https://pennstate.service-now.com/sp?id=sc_cat_item&sys_id=dd1c98f11b57e510bd31ed74bd4bcb1a)
+- **Support Ticket** : Submit through the Roar portal
+
+Our team of HPC experts will respond as quickly as possible.
+
+**Important**: In your request, **clearly state which Roar resource** your issue concerns:
+- Roar Collab (RC)
+- Roar Restricted (RR)
+
+This helps us route your request to the right specialist and resolve it faster.
+
+**Never** email staff members directly. Using the official channels ensures:
```

Comment:
Let's reword this to be a bit softer. Something like:

It is preferable to send in a support request using the above channels versus emailing staff directly. This benefits you by: